pkg/generated/openapi/def.bzl

```diff
@@ -0,0 +1,70 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_kubernetes_build//defs:go.bzl", "go_genrule")
+
+OPENAPI_TARGETS = [
+    "cmd/libs/go2idl/client-gen/test_apis/testgroup/v1",
+    "cmd/libs/go2idl/openapi-gen/generators",
+    "federation/apis/federation/v1beta1",
+    "pkg/api/v1",
+    "pkg/apis/abac/v0",
+    "pkg/apis/abac/v1beta1",
+    "pkg/apis/apps/v1beta1",
+    "pkg/apis/authentication/v1",
+    "pkg/apis/authentication/v1beta1",
+    "pkg/apis/authorization/v1",
+    "pkg/apis/authorization/v1beta1",
+    "pkg/apis/autoscaling/v1",
+    "pkg/apis/autoscaling/v2alpha1",
+    "pkg/apis/batch/v1",
+    "pkg/apis/batch/v2alpha1",
+    "pkg/apis/certificates/v1beta1",
+    "pkg/apis/componentconfig/v1alpha1",
+    "pkg/apis/extensions/v1beta1",
+    "pkg/apis/imagepolicy/v1alpha1",
+    "pkg/apis/policy/v1beta1",
+    "pkg/apis/rbac/v1alpha1",
+    "pkg/apis/rbac/v1beta1",
+    "pkg/apis/storage/v1beta1",
+    "pkg/version",
+]
+
+OPENAPI_VENDOR_TARGETS = [
+    "k8s.io/apimachinery/pkg/api/resource",
+    "k8s.io/apimachinery/pkg/apis/meta/v1",
+    "k8s.io/apimachinery/pkg/runtime",
+    "k8s.io/apimachinery/pkg/util/intstr",
+    "k8s.io/apimachinery/pkg/version",
+    "k8s.io/apiserver/pkg/apis/example/v1",
+    "k8s.io/apiserver/pkg/server/openapi",
+    "k8s.io/client-go/pkg/api/v1",
+]
+
+
+def openapi_library(name, tags, srcs):
+  deps = [
```

Comment:
No, this file is hand-written. However, gazel will still automanage the `go_default_library` in `pkg/generated/openapi/BUILD`. This isn't a big deal for this generated file, since its only source is `doc.go`, but it will be important for the other `zz_generated` files.